packages/chakra-ui-core/src/CTooltip/CTooltip.js

```diff
@@ -204,6 +204,8 @@ const CTooltip = {
       }
     }
 
+    console.log(this.$attrs)
```

Comment:
console.log

Reply:
Still debugging like the good old days 😂